src/bootstrap.js

```diff
@@ -170,7 +183,7 @@ export class Bootstrap {
     plugins = this.resolve(plugins);
 
     // Install in order
-    plugins.forEach((name) => this.__install(name));
+    each(plugins, (name) => this.__install(name))
```

Comment:
**Bugfix TLDR**: [`Array.prototype.forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) is usually, but not quite always, a replacement for `_.each`.

This line [originally used](https://github.com/unconed/threestrap/pull/12/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-4751ae25393146c4568345c10bab05f6ad2fad18119b0dd65ed7de6e09fbca92L149-R156) lodash's `_.each`, which is an alias for [`_.forEach`](https://lodash.com/docs/4.17.15#forEach). Lodash's `_.forEach` allows for bailing out of the loop early by returning `false`.  Thus if a plugin returns `false` from its install method, the overall installation process is aborted. (The "fallback" plugin uses this approach; unsure whether others do, too).

The native [`Array.prototype.forEach()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) has no such early-bail. 😦 

Note: In `_.each` / `_.forEach`, only a literal `false` causes early bail. Other falsy values (like undefined) do not. So functions that have no explicit return value will never trigger an early bail.

Reply:
Really good to know. I had another nightmare error from my port away from lodash, iterating over an array... can't remember the details now but thanks for finding this one and explaining it.